### PR TITLE
Dinucleotide patterns analysis tools v1.0

### PR DESCRIPTION
**Pull request by erinijapranckeviciene**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/dnp-binstrings/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  sha256: 19c46d6bc6a1af70e5a2b4a051aab334026dc38c84b367ea9c445da3fdb21cfe
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - seqan-library
```
Comment (bgruening) [inline recipes/dnp-binstrings/meta.yaml]: Seqan should probably in the host section.

Comment (erinijapranckeviciene) [reply recipes/dnp-binstrings/meta.yaml]: Thank you for the comment. It was a journey... No. Seqan is C++ header only library. It matters only in compilation.  But it depends on zlib and bzip2. The two should be in the host and run.

**Comment by bgruening** [comment]
Thanks @erinijapranckeviciene!

**Comment by erinijapranckeviciene** [comment]
Thank you @bgruening !  Does it mean I can try to install my tools with conda install?

**Comment by bgruening** [comment]
Yes. You can see it here: https://anaconda.org/bioconda/ :)

**Comment by erinijapranckeviciene** [comment]
Thank you. This is only half of the journey.  Now I will be submitting tools that depend on the dnpatterntools to the Galaxy ToolShed.  Thank you and bioconda team for being a driving force of bioinformatics open source  community. With best regards !

**Comment by bgruening** [comment]
@erinijapranckeviciene your are welcome!

Looking at the bioconda page, your packages are not named `dnp-*`. This is wrong isn't it? They should be labeled with dnp as prefix I guess. If so please create a PR with `name: ` change to include the prefix and I will take care of the rest.

**Comment by erinijapranckeviciene** [comment]
Thank you for the notice. I don't know if I am breaking some naming rules
with this naming. The packages don't need the dnp- prefix because they are
individual packages. I made dnp- subdirs for myself to be able to search
them as a group in the recipes directory and  did not rename in PR. Shall I
drop this dnp- subdir prefix and create a pull request to make subdirs
named after the packages to have things tidy, or it  is OK to have them
like this?

**Comment by bgruening** [comment]
I would slightly prefer the prefix solution. As your names are a little bit generic and I fear name-collisions with other recipes/programs.

**Pull request merged**